gapic/src/main/com/google/gapid/widgets/Widgets.java

```diff
@@ -408,6 +408,14 @@ public static Button createButton(Composite parent, String text, Listener listen
     return result;
   }
 
+  public static Button createButton(Composite parent, String text, int style, Color color, Listener listener) {
+    Button result = new Button(parent, style);
```

Comment:
I think the style should be or'ed **here** with `SWT.PUSH`. I see usage only with `SWT.FLAT`, which is rendering hint, and so the buttons are created without the type style and it defaults to push buttons. To be explicit, what you want is `SWT.PUSH | SWT.FLAT`, but with these factory methods, we've removed the `SWT.PUSH` vs. `SWT.CHECK` vs. `SWT.RADIO` non-sense from the callers, so we should stay with that and only use it to specify hint styles.

Reply:
Done, thanks for explaining, now added it to make it explicit and conform with other factory functions.